code/modules/mob/living/carbon/carbon.dm

```diff
@@ -171,6 +171,8 @@
 		var/obj/item/thrown_item = thrown_thing
 		if(thrown_item.throw_verb)
 			verb_text = thrown_item.throw_verb
+	do_attack_animation(target, no_effect = 1)
+	playsound(loc, 'sound/weapons/punchmiss.ogg', 50, TRUE, -1)
```

Comment:
```suggestion
	playsound(src, 'sound/weapons/punchmiss.ogg', 50, vary = TRUE, extrarange = -1)
```
- `src` or `loc` doesn't matter, the former is more clear though
- named args

Reply:
Also just a thought but I wonder if pitching the sound effect based on how strong the throw is (power and range) would be neat.

Reply:
I will try fiddling with the values and see if there is a satisfying setup.